modules/accounts/accounts.js

```diff
@@ -1,191 +1,259 @@
         } catch(error) {
-            return Helper.outputError(` occured while restroing account : ${error}`);
+            Logger.error(`Error occured while restroing account : ${error}`);
         }
     }
 
     async activateAlphanetAccount(account) {
+        const conseiljs = require(CONSEIL_JS);
+        const tezosNode = config.provider;
+        let conseilServer = { 'url': CONSEIL_SERVER_URL, 'apiKey': CONSEIL_SERVER_APIKEY, 'network': TESTNET_NAME };
         const keys = this.getKeys(account);
+
+        if(Helper.confirmNodeProvider(tezosNode)) {
+            Logger.error('Make sure your current provider is set to remote node provider.');
+            return;
+        }
+
         if(!keys || !keys.secret) {
-            return Helper.outputError(`Couldn't find keys for given account.\nPlease make sure the account exists and added to tezster.`);
+            Logger.error(`Couldn't find keys for given account.\nPlease make sure the account exists and added to tezster.`);
+            return;
         }
+        const keystore = {
+            publicKey: keys.pk,
+            privateKey: keys.sk,
+            publicKeyHash: keys.pkh,
+            seed: '',
+            storeType: conseiljs.StoreType.Fundraiser
+        };
+
         try {
-            let activationResult = await eztz.rpc.activate(keys.pkh, keys.secret);
-            return Helper.output(`successfully activated testnet faucet account: ${keys.label} : ${keys.pkh} \n with tx hash: ${activationResult}`);
+            Logger.warn('Activating the account....');
+            let activationResult = await conseiljs.TezosNodeWriter.sendIdentityActivationOperation(tezosNode, keystore, keys.secret);
+            
+            const activationGroupid = this.clearRPCOperationGroupHash(activationResult.operationGroupID);
+            await conseiljs.TezosConseilClient.awaitOperationConfirmation(conseilServer, conseilServer.network, activationGroupid, 10, 30+1);
+
+            const revealResult = await conseiljs.TezosNodeWriter.sendKeyRevealOperation(tezosNode, keystore);
+            Logger.info(`Testnet faucet account successfully activated: ${keys.label} - ${keys.pkh} \nWith tx hash: ${JSON.stringify(revealResult.operationGroupID)}`);
         } catch(error) {
-            return Helper.outputError(` occured while activating account : ${error}`);
+            Logger.error(`Error occured while activating account : ${error}`);
         }
     }
 
+    async deleteAccount(account) {
+        const keys = this.getKeys(account);
+
+        if(!keys) {
+            Logger.error(`Couldn't find keys for given account.\nPlease make sure the account exists and added to tezster. Run 'tezster list-accounts' to get all accounts`);
+            return;
+        }
+
+        if(keys.label.includes('bootstrap1' || 'bootstrap2') || keys.label.includes('bootstrap3') || keys.label.includes('bootstrap4') || keys.label.includes('bootstrap5')) {
```

Comment:
Do not repeat your self like checking for all bootstrap account with 5 conditions.
/bootstrap[1-5]/ use this regex pattern and match with it to find all bootstrap account.